gradio/chat_interface.py

```diff
@@ -123,24 +127,37 @@ def __init__(
                             container=False,
                             show_label=False,
                             placeholder="Type a message...",
-                            scale=10,
+                            scale=7,
                         )
                     if submit_btn:
                         if isinstance(submit_btn, Button):
                             submit_btn.render()
                         elif isinstance(submit_btn, str):
                             submit_btn = Button(
-                                submit_btn, variant="primary", scale=1, min_width=0
+                                submit_btn, variant="primary", scale=1, min_width=150
                             )
                         else:
                             raise ValueError(
                                 f"The submit_btn parameter must be a gr.Button, string, or None, not {type(submit_btn)}"
                             )
-                    self.buttons.append(submit_btn)
+                    if stop_btn:
+                        if isinstance(stop_btn, Button):
```

Comment:
We can check if the button `visible` prop is true here and raise an error if not

Reply:
Updated to just set the prop to False

Reply:
Let me do this for a few other components as well actually